lib/ansible/plugins/filter/encryption.py

```diff
@@ -17,19 +17,26 @@
 display = Display()
 
 
-def do_vault(data, secret, salt=None, vaultid='filter_default', wrap_object=False):
+def do_vault(data, secret, salt=None, vault_id='filter_default', wrap_object=False, vaultid=None):
```

Comment:
@obaranov this should fix your issue #81420.

Reply:
^ already linked in sumary